src/Compilers/VisualBasic/Portable/Binding/Binder_Expressions.vb

```diff
@@ -91,7 +91,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     ' parent expression.
 
                     ' Dev10 allows parenthesized type expressions, let's bind as a general expression first.
-                    Dim operand As BoundExpression = BindExpression(DirectCast(node, ParenthesizedExpressionSyntax).Expression, False, False, eventContext, diagnostics)
+                    Dim operand As BoundExpression = BindExpression(DirectCast(node, ParenthesizedExpressionSyntax).Expression,
+                                                                    isInvocationOrAddressOf:=False,
```

Comment:
>False [](start = 93, length = 5)

should we pass `isInvocationOrAddressOf` here?

Reply:
> should we pass isInvocationOrAddressOf here?

I do not think so and that wouldn't have any affect of the scenarios this change is targeting.

---
In reply to: [328824413](https://github.com/dotnet/roslyn/pull/38802#discussion_r328824413) [](ancestors = 328824413)